lib/stale.js

```diff
@@ -33,6 +40,12 @@ module.exports = class Stale {
     const days = this.config.days || this.config.daysUntilClose
     return this.search(days, query)
   }
+  
+  getLockableIssues () {
+    const query = `label:"${this.config.staleLabel}"`
```

Comment:
I'm just trying to think through how this will work. If I have `daysUntilClose: 30` and `daysUntilLock: 7`, then this will lock the issue before it's closed. So it probably needs `is:closed` as part of the query too.

Reply:
That's a good point. I'll do that